test/unit/commands.test.ts

```diff
@@ -109,3 +109,14 @@ describe('class OpCompressedRequest', () => {
     }
   });
 });
+
+describe('OpMsgRequest', () => {
+  describe('fire-and-forget', () => {
```

Comment:
```suggestion
  describe('#constructor', () => {
```
Since we are describing behaviour inside the class' constructor.